pkg/api/graphql/resolvers/mutations.go

```diff
@@ -65,7 +65,17 @@ func (r *MutationResolver) CreateEvent(args struct{ Event EventInput }) (graphql
 		return "", err
 	}
 
-	r.msgProducer.Async(message.NewEvent(event))
+	r.msgProducer.Async(message.NewEvent(*event))
+
+	eventReceiverGroups, err := storage.FindTriggeredEventReceiverGroups(r.Connection.Client, *event, event.EventReceiverID)
+	if err != nil {
+		logger.Error(err, "error finding triggered event receiver groups", "input", eventInput)
+		return "", err
+	}
+
+	for _, eventReceiverGroup := range eventReceiverGroups {
+		r.msgProducer.Async(message.NewEventReceiverGroupComplete(*event, eventReceiverGroup))
+	}
```

Comment:
This change and the one in event.go are duplicates. Can we combine them into one function?

Reply:
I'd say keep duplicated for now, this can be handled along with #70 . Agreed though, there should be a common place for logic such as validation & resource creation